src/components/Person/Person.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
prop-types seems to not be pre-installed in this project, so you also should push the package.json to pass the tests on github